chia/consensus/block_creation.py

```diff
@@ -215,14 +212,19 @@ def create_foliage(
                     )
                     reward_claims_incorporated += [pool_coin, farmer_coin]
                     curr = blocks.block_record(curr.prev_hash)
-        additions.extend(reward_claims_incorporated.copy())
-        for coin in additions:
+
+        for coin in reward_claims_incorporated:
```

Comment:
Rewards are now first, does the ordering change matter?

Reply:
I'm pretty sure the order does not matter. the BIP158 filter is deterministic